system/settings/settings.xml

```diff
@@ -49,6 +49,9 @@
         <setting id="videoplayer.usedisplayasclock" type="boolean" label="13510" help="36166">
           <level>1</level>
           <default>false</default>
+          <dependencies>
+            <dependency type="enable" setting="audiooutput.passthrough" operator="is">false</dependency>
```

Comment:
I'd also hadn't checked the upgrade scenario where both are already enabled, thinking about it that could cause problems with this anyway.